Add Option to Specify Request Timeout in PagerDuty API Session. (#993)
lyft/cartography: docs/root/modules/pagerduty/config.md

1. Prepare your PagerDuty API key.
    1. Generate your API token by following the steps from the PagerDuty [Generating API Keys documentation](https://support.pagerduty.com/docs/generating-api-keys)
    1. Populate an environment variable with the API key. You can pass the environment variable name via CLI with the `--pagerduty-api-key-env-var` parameter.

>     1. Populate an environment variable with the API session timeout. You can also set this via CLI with `--pagerduty-request-timeout` parameter.

Reviewer: ```suggestion
    1. You can set the timeout for pagerduty requests via the CLI with `--pagerduty-request-timeout` parameter.
```
